src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { useState } from 'react';
+import './createCourse.css';
+import Input from '../../common/Input/Input';
+import Button from '../../common/Button/Button';
+import time_convert from '../../helpers/pripeDuration';
+// import { mockedAuthorsList } from '../../constants';
+
+const CreateCourse = ({
+	authors,
+	setAuthors,
+	courses,
+	setCourses,
+	isShown,
+	setIsShown,
+}) => {
+	// Add Author
+	const [addAuthor, setAddAuthor] = useState({
+		name: '',
+	});
+
+	const [allAuthors, setAllAuthors] = useState(authors);
+
+	const handleAddFormChange = (event) => {
+		event.preventDefault();
+
+		const fieldName = event.target.getAttribute('name');
+
+		const fieldValue = event.target.value;
+
+		const newFormData = { ...addAuthor };
+		newFormData[fieldName] = fieldValue;
+
+		setAddAuthor(newFormData);
+	};
+
+	const handleAddFormSubmit = (event) => {
+		event.preventDefault();
+		const name = addAuthor.name;
+		if (name === '') {
+			alert('Input name cant be empty');
+		} else if (name.length <= 2) {
+			alert('Must contain 2 or more char');
+		} else {
+			const newAuthor = {
+				id: Math.random() + '',
+				name: name,
+			};
+
+			const newAuthors = [...authors, newAuthor];
+			setAllAuthors([...allAuthors, newAuthor]);
+			setAuthors(newAuthors);
+		}
+
+		const inputField = document.querySelector('.input-author');
+		console.log(inputField);
+		inputField.value = '';
+	};
+	console.log(allAuthors);
+
+	// Add duration
+	const [duration, setDuration] = useState(0);
```

Comment:
state-ove deklarisemo na pocetku same komponente, to je konvencija koja se trenutno prati